src/ducks/project.js

```diff
@@ -66,7 +66,7 @@ const classifierReducer = (state = initialState, action) => {
 
 //Action Creators
 
-const fetchProject = (id = TEMPORARY_HARDCODED_PROJECT_ID) => {
+const fetchProject = (id = config.zooniverseLinks.projectId) => {
```

Comment:
Ahh, nice. I made a similar change in https://github.com/zooniverse/liberating-the-liberator/pull/88, but I'll remove that fix in lieu of this after the merge.